packages/features/ee/workflows/lib/reminders/reminderScheduler.ts

```diff
@@ -48,7 +48,7 @@ const processWorkflowStep = async (
 ) => {
   if (isSMSAction(step.action)) {
     await checkSMSRateLimit({
-      identifier: `sms:${workflow.userId ? "user:" : "team:"}${workflow.userId || workflow.teamId}`,
```

Comment:
Also team workflows have a `userId` (the user that created the workflow). We need to check for `teamId` here